e2e/ruby/rails/demo/Gemfile.lock

```diff
@@ -1,7 +1,7 @@
 PATH
   remote: ../../../../sdk/highlight-ruby/highlight
   specs:
-    highlight_io (0.4.1)
+    highlight_io (0.5.1)
```

Comment:
This is just the gemfile for the E2E demo app, so I don't think it's used anywhere. I guess it wasn't upgraded during the previous release, but the latest changes were being reflected when click testing locally because it is using a file path for the gem.

This was only updated because I `bundle install`ed the gem from rubygems.org to repro the error that was reported since I couldn't repro it when referencing the gem via the local file path. Re-ran `bundle install` after changing it back and committing my changes.